scimath/Functionals/Function.h

```diff
@@ -219,14 +219,6 @@ class RecordInterface;
      template <class W, class X>
      Function(const Function<W,X> &other) : param_p(other.parameters()),
      arg_p(0), parset_p(other.parsetp()), locked_p(False) {}
-     Function(const Function<T,U> &other) :
```

Comment:
Why does this disappear?

Reply:
Same reason as before, this copy constructor causes warnings, the default is fine.